plugins/habitat_plugin/habitat_sensors.py

```diff
@@ -160,10 +165,11 @@ def get_observation(
 
 class TargetObjectSensorHabitat(Sensor[HabitatEnvironment, PointNavTask]):
     def __init__(self, uuid: str = "target_object_id", **kwargs: Any):
-        observation_space = gym.spaces.Discrete(38)
-
```

Comment:
I think we're missing a `observation_space = self._get_observation_space()` call here? Otherwise nothing is getting passed to the super call.

Reply:
Wow, I tried my best to cover all of them, but I missed those two. Thanks for the check!